src/apikey/lib/routers/admin.js

```diff
@@ -2,6 +2,14 @@ const router = require('express').Router();
 
 const checkAuth = require('../middlewares/auth');
 
-router.post('/login', checkAuth, async (req, res, next) => res.status(204).json());
+/**
+ * Route that checks if the content of the x-api-key header
+ * matches the environment variable used as password.
+ *
+ * @param {import('express').Request} req - HTTP request.
+ * @param {import('express').Response} res - HTTP response.
+ * @return {import('express').Response} HTTP response.
+ */
+router.post('/login', checkAuth, async (req, res) => res.status(204).json());
```

Comment:
This comment describes the router `post()` call, but the parameters are those of the `handler` function. This notation may be incorrect.